tools/build_pytorch_libs.sh

```diff
@@ -315,6 +319,7 @@ function build_caffe2() {
 		       -DUSE_LEVELDB=$USE_LEVELDB \
 		       -DUSE_LMDB=$USE_LMDB \
 		       -DUSE_OPENCV=$USE_OPENCV \
+           -DUSE_QNNPACK=$USE_QNNPACK \
```

Comment:
Nit: Same spaces in front.

Reply:
formatting in this file is all over the place